src/sparql/TripleBuilder.ts

```diff
@@ -2,13 +2,15 @@ import { Condition } from '@/sparql/QueryRepresentation';
 import { IriTerm, PropertyPath, Term, Triple } from 'sparqljs';
 import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 import rdfNamespaces from '@/sparql/rdfNamespaces';
+import ReferenceRelation from '@/data-model/ReferenceRelation';
 
 export default class TripleBuilder {
 	public buildTripleFromQueryCondition( condition: Condition ): Triple {
 		return {
 			subject: {
 				termType: 'Variable',
-				value: 'item',
+				value: condition.referenceRelation === ReferenceRelation.Regardless ?
+					'item' : 'statement' + condition.propertyId,
```

Comment:
Technically it's correct but we can have a property being repeated more than once (occupation matching artist and occupation matching engineer) so maybe adding condition index would be better?

Reply:
Yes, agree. I mentioned it at the top of the conversation here. That's one of the things that are still missing. Thanks for the suggestion.